jazzy.sh

```diff
@@ -0,0 +1,53 @@
+osName="osx"
```

Comment:
We should update the logic in this file (`jazzy.sh`). Looking at the current contents of `jazzy.sh`, there is quite a bit code that I don't think is needed. For instance, I don't think lines 1 - 37 are needed. The Swift binaries are already installed by the time we invoke `jazzy.sh`. What we need instead (in `jazzy.sh`) is something along these lines:

1) Verify that a `.jazzy.yaml` does exist in the root folder of the repo. For instance, see https://github.com/IBM-Swift/CloudEnvironment/blob/master/.jazzy.yaml.
2) If `.jazzy.yaml` does **not** exist, then we do not attempt to generate jazzy docs.
3) If `.jazzy.yaml` **does** exist, then we attempt to generate the docs:
- Install jazzy
- Generate xcode project (see comments below)
- Run jazzy
- Once we have generated the jazzy docs, we should then commit those changes and push them to the corresponding branch of the repo. Note that we don't want to create an infinite build loop, which requires us to use a git command that looks like this git commit -m 'documentation update [ci skip]'. @ddunn2 can give you more details on this.

Now, regarding the logic for generating xcode project, we should leverage the code we already have at https://github.com/IBM-Swift/Package-Builder/blob/1294bef1ffd89d74536ce06fab97a1c0c4919a37/codecov.sh#L35. However, let's avoid code duplication (aka copy and paste). :-) In order to leverage that code, we should do a little bit of refactoring of the existing code found in `codecov.sh`. This way we can use the same code for generating the xcode project from both places, `codecov.sh` and `jazzy.sh`. This implies extracting the Xcode project code generation to a separate file so we can then call it from `codecov.sh` and `jazzy.sh`.